lib/Table.js

```diff
@@ -40,6 +40,7 @@ const compareIndexes = function compareIndexes(local, remote) {
         let localIndex = (({ IndexName, KeySchema, Projection }) => ({ IndexName, KeySchema, Projection }))(localIndexes[i]);
         let remoteIndex = (({ IndexName, KeySchema, Projection }) => ({ IndexName, KeySchema, Projection }))(remoteIndexes[j]);
 
+        debug("\nindex being compared \nlocal:", localIndex, "\nremote: \n", remoteIndex)
```

Comment:
What are the thoughts about changing this to one of the following options? Not sure if this is a good idea, and I'm not sure what we are doing in cases like this in other areas.

Not sure if I'm the biggest fan of those `\n` in the string. Especially since I'm not sure how it actually looks in the console because debug appends that string and other information to the beginning of every debug statement. So I'm not sure if it'd look really weird with those line breaks.

```
debug("index being compared, local:", localIndex, ", remote: ", remoteIndex)
```

```
debug("index being compared");
debug("local: ", localIndex);
debug("remote: ", remoteIndex);
```

Not sold on my ideas, or that you are doing it wrong. Maybe your way is the best. Just wondering what your thoughts are in terms of this.

If you could also look throughout the project and see if you can find other areas where we are doing `\n` or one of my methods, that'd be awesome too.

Reply:
I actually prefer it your way. was just hacking it out to see if it worked when I wrote that lol. fixing it update